components/org.wso2.carbon.appmgt.hostobjects/src/main/java/org/wso2/carbon/appmgt/hostobjects/APIProviderHostObject.java

```diff
@@ -297,7 +299,8 @@ public static void jsFunction_deleteBusinessOwner(Context context, Scriptable th
         }
         String ownerId = args[0].toString();
         APIProvider apiProvider = getAPIProvider(thisObj);
-        apiProvider.deleteBusinessOwner(ownerId);
+        String deletedOwnerId = apiProvider.deleteBusinessOwner(ownerId);
```

Comment:
**apiProvider.deleteBusinessOwner** this method throws AppManagementException. We should log it in here.